src/Cake.Common.Tests/Fixtures/Tools/OctopusDeployPusherFixture.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System.Collections.Generic;
+using Cake.Common.Tools.OctopusDeploy;
+using Cake.Core.IO;
+using Cake.Testing.Fixtures;
+
+namespace Cake.Common.Tests.Fixtures.Tools
+{
+    internal sealed class OctopusDeployPusherFixture : ToolFixture<PushPackageSettings>
+    {
+        public List<FilePath> Packages { get; set; }
```

Comment:
@agc93 I was referring to this:

![image](https://cloud.githubusercontent.com/assets/1271146/16149793/078b3270-348b-11e6-901d-fc187167ec36.png)

i.e. new line between property and constructor.